packages/react-core/src/components/Label/Label.tsx

```diff
@@ -79,11 +92,14 @@ export const Label: React.FunctionComponent<LabelProps> = ({
         className
       )}
     >
-      <Component className={css(styles.labelContent)} {...(href && { href })}>
-        {icon && <span className={css(styles.labelIcon)}>{icon}</span>}
-        {isTruncated && <span className={css(styles.labelText)}>{children}</span>}
-        {!isTruncated && children}
-      </Component>
+      {customContent ? (
+        <>{customContent}</>
```

Comment:
are we able to add `css(styles.labelContent)` to whatever is passed in?

Reply:
yes if the code is changed to clone the child and apply our classname (not recommended) or better to use renderprops and pass that classname back to the user so they can apply it (see my example below)

Reply:
@mcoker I am trying Joachim's suggestion below.